app/_src/policies/meshfaultinjection.md

```diff
@@ -0,0 +1,253 @@
+---
+title: MeshFaultInjection (beta)
+---
+
+With the MeshFaultInjection policy you can easily test your microservices against resiliency.
+
+{% warning %}
+This policy uses a new policy matching algorithm and is in beta state. It should not be combined with [FaultInjection](/docs/{{ page.version }}/policies/fault-injection).
+{% endwarning %}
+
+`MeshFaultInjection` policy is available only for L7 HTTP traffic,
+therefore `kuma.io/protocol` is a mandatory tag to properly propagate configuration.
+
+## `targetRef` support matrix
+
+| `targetRef.kind`    | top level | to  | from |
+| ------------------- | --------- | --- | ---- |
+| `Mesh`              | ✅        | ❌  | ✅   |
+| `MeshSubset`        | ✅        | ❌  | ✅   |
+| `MeshService`       | ✅        | ❌  | ✅   |
+| `MeshServiceSubset` | ✅        | ❌  | ✅   |
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+`MeshFaultInjection` allows configuring a list of HTTP faults. They execute in the same order as they were defined.
```

Comment:
Maybe we should mention that every next item in `http` array has a percentage that depends on previous items. We can show that:
```yaml
http:
  - abort:
      httpStatus: 500
      percentage: 70
  - abort:
      httpStatus: 503
      percentage: 50
```
503 will appear in `0.3 * 0.5 * 100` cases because each item is a filter in the filter chain